sora-android-sdk/src/main/kotlin/jp/shiguredo/sora/sdk/channel/SoraMediaChannel.kt

```diff
@@ -86,7 +87,13 @@ class SoraMediaChannel @JvmOverloads constructor(
     ignoreDisconnectWebSocket: Boolean? = null,
     dataChannels: List<Map<String, Any>>? = null,
     private var bundleId: String? = null,
+    @Deprecated(
+        "2025 年 12 月に廃止します。",
```

Comment:
SDK 側の廃止時期は未定なので、例えば以下のように書いておいてもらえますか。他も同様です。
```suggestion
        "この項目は 2025 年 12 月リリース予定の Sora にて廃止されます",
```

Reply:
こちらこのように変更していきます（ほかも